microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-examples.adoc

```diff
@@ -1,10 +1,25 @@
 = Azure Service Bus Connector 3.0 Examples - Mule 4
- 
-// Add text to introduce the example section and say what one finds 
-// here and why it is helpful or necessary info.
 
-// Put Examples here -- You may need to obtain these from the 
-// code repo /demo folder.
+There are demos available that you can download and configure for your own environment to see different flows that use the Azure Service Bus Connector operations. 
+
+== Import the Demo Projects
+
+Download the `demo` folder and import the demo projects into Anypoint Studio.  
+
+. Go to `https://github.com/mulesoft/mule4-azure-service-bus-messaging-connector` and download the connector as a ZIP file.
+. Extract the files from the downloaded ZIP file.
+. In Studio, go to *File > Import*.
+. Select *Anypoint Studio > Anypoint Studio Project from File System* and click *Next*.
+. In *Project Root*, navigate to the `<download-location>-mule4-azure-service-bus-messaging-connector/demo/` and select it.
+. In *Server runtime*, select *Mule Server 4.2.x* (or later).
```

Comment:
Maybe this is just me, but it seems a bit odd to quote a UI label selection and then provide an alternative "(or later)".
What about...:
```suggestion
. In *Server runtime*, select your server version, which must be 4.2.x or later.
```